### PR TITLE
Add Italian translation of create_model.mdx  and serialization.mdx 

### PR DESCRIPTION
**Pull request by F02934**
# What does this PR do?
Italian translation of create_model.mdx and serialization.mdx
Issues : #17459 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.
@mfumanelli

### CODE REVIEW

**Comment by mfumanelli** [comment]
Hi @F02934 ❤️ Thanks for your contribution! The reason why the code does not pass the tests is related to the fact that the file you create with the translation must be called exactly by the name in the toctree under "local:", in this case in local it says create_a_model while your file is create_model.

While I'm at it, I'd like to ask you if you can also translate the part in the _toctree called "title" for example with: "Crea un'architettura personalizzata". I will now take a moment to look at the rest of the PR, thank you 🌈🌈🌈

**Comment by F02934** [comment]
Alright I will correct it!

**Comment by F02934** [comment]
@mfumanelli Done!:)

**Comment by mfumanelli** [comment]
Hi @F02934, I have just had a look at your file, I ask you if you can edit these things:
- "Crea un architettura personalizzata" --> "Crea un'architettura personalizzata"
- "chiunque sia interessato nel studiare" --> "qualunque persona sia interessata nel studiare" (to keep the text more gender-neutral)
- "Creare un architettura modello" --> "Creare un'architettura per un modello"
- "Creare un tokenizer per testo lento e veloce" --> "Creare un tokenizer lento e veloce per il testo"
- "Una configuration si riferisce agli" --> "Una configurazione si riferisce agli"
- "Dai un'occhiata piu vicino a" --> "Dai un'occhiata più da vicino a"
- "creando uno spazion per sperimentare" --> "creando uno spazio per sperimentare"
- "Utilizzare tasso di drop out più elevato per le probalità di attenzione con il parametro attention_dropout" --> "Utilizzare un tasso di dropout più elevato per le probabilità di attention con il parametro attention_dropout"
- "la configurazione del modello ti soddisfa, lo puoi salvare con" --> "la configurazione del modello ti soddisfa, la puoi salvare con"
- "Il file della tua configurazione e memorizzato" --> "Il file della tua configurazione è memorizzato"
- "Il prossimo passo e di creare model." --> "Il prossimo passo è creare un modello."
- "sono usati per definire l'archittetura" --> "sono usati per definire l'architettura"
- "soppressione dei self-attention heads" --> "soppressione delle self-attention heads"
- "Questo significa che i modelli sono compatibili con ciascuno dei rispettivi utilizzi del framework" --> "Ciò significa che i modelli sono compatibili con l'uso di ciascun framework."
- "la configurazione del modello predefinito è automaticamente caricato" --> "la configurazione del modello predefinito è automaticamente caricata"
- "puoi ancora sostituire - alcuni o tutti - gli attributi di configurazion del modello predefinito con i tuoi se lo desideri:" --> "puoi ancora sostituire gli attributi - alcuni o tutti - di configurazione del modello predefinito con i tuoi se lo desideri:"
(these last corrections above for both pytorch and tf)
- edit this everywhere "modello head" with "model head"
- "hai un modello DistilBERT base in cui output sono gli hidden states" --> "hai un modello DistilBERT base i cui output sono gli hidden states"
- "DistilBERT base con la sequenza di classificazione head" --> "DistilBERT base con una testa di classificazione per sequenze"
- edit this everywhere "sequenza di classificazione head" with "testa di classificazione per sequenze"
- "per un altra attività" --> "per un'altra attività"
- "ad un modello head diverso" --> "ad una model head differente"
- "Il head di risposta alle domande" --> "La head per compiti di question answering"
- "un occhiata a questo table" --> "un'occhiata a questa tabella"
- "tuo file vocabulario:" --> "tuo file vocabolario:"
- "Per impostazione predefinita" --> "Per l'impostazione predefinita"

thank you very much! 🚀🚀

**Comment by F02934** [comment]
hello @mfumanelli I edited the things you asked, you can check if everything is alright. I also made a mistake and commited here  another translation I was doing but it's small (I added the serialization on _toctree.yml and mistakenly commited here). What should I do?

**Comment by mfumanelli** [comment]
Hi @F02934! You can simply remove the file from your repo locally and re-commit and push. Or if you prefer, add "WIP" to the PR title and also add to the PR description that you are translating the serialization file 😊

**Comment by F02934** [comment]
Alright I will remove the file from my local repo and re-commit push

**Comment by F02934** [comment]
@mfumanelli everything good know! You can check if there is any problems

**Comment by F02934** [comment]
Hello @mfumanelli I finished another translation (serialization.mdx) But I don't know why it keep pushing to this pull request.

**Comment by F02934** [comment]
I changed the PR describe that I also translated serialization.mdx here

**Comment by mfumanelli** [comment]
Hi @F02934!! I was just about to reply to you! Your PR keeps updating because you are working on the main of the forked repo. If you create a branch for each file you translate, this doesn't happen. I'll also look at the new file later, sorry I didn't reply sooner.

**Comment by F02934** [comment]
@mfumanelli ok sorry I'm still new thank you for answering. So I need to create another branch each time I'm working on a new translation rigth?

**Comment by F02934** [comment]
Hello @mfumanelli as you know from our conversation I didn't make a branch for these two translation. I noticed a made a mistake with toctree (maybe) how do I make changes here. Because now I'm starting already started to work on other things. How do I commit to make changes here again?

File: docs/source/it/_toctree.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@
 - sections:
   - local: create_a_model
     title: Crea un'architettura personalizzata
+  - sections:
   - local: serialization
     title: Esporta modelli Transformers
-  title: How-to guides
```
Comment (F02934) [inline docs/source/it/_toctree.yml]: `- sections:
  - local: fast_tokenizers
    title: "Use tokenizers from 🤗 Tokenizers"
  - local: create_a_model
    title: Create a custom architecture
  - local: custom_models
    title: Sharing custom models
  - sections:
    - local: tasks/sequence_classification
      title: Text classification
    - local: tasks/token_classification
      title: Token classification
    - local: tasks/question_answering
      title: Question answering
    - local: tasks/language_modeling
      title: Language modeling
    - local: tasks/translation
      title: Translation
    - local: tasks/summarization
      title: Summarization
    - local: tasks/multiple_choice
      title: Multiple choice
    - local: tasks/audio_classification
      title: Audio classification
    - local: tasks/asr
      title: Automatic speech recognition
    - local: tasks/image_classification
      title: Image classification
    title: Fine-tune for downstream tasks
  - local: run_scripts
    title: Train with a script
  - local: sagemaker
    title: Run training on Amazon SageMaker
  - local: multilingual
    title: Inference for multilingual models
  - local: converting_tensorflow_models
    title: Converting TensorFlow Checkpoints
  - local: serialization
    title: Export 🤗 Transformers models
`
This is the englis toctree.yml the changes I made are good or not?

**Comment by F02934** [comment]
@mfumanelli hi , just wondering if you forgot about this pr 🧑‍💻

**Comment by mfumanelli** [comment]
Hi @F02934, no! It will be approved, there are several PRs in the stack, don't worry <3

**Comment by LysandreJik** [comment]
cc @omarespejel

**Comment by nickprock** [comment]
Hi @F02934 could you solve conflicts, please?
I think they are in the _toctree.yml because some PRs have been merged and the file has updated.

> > cc @omarespejel
> 
> Hi what does it mean?

@omarespejel and @sgugger are the reviewers with write access to the repo.

**Review by sgugger — APPROVED** [review]
Thanks a lot for translating those two files!
